bodhi/tests/client/test_bindings.py

```diff
@@ -1316,3 +1426,44 @@ def test_testable_no_dnf(self):
             list(client.testable())
 
         self.assertEqual(str(exc.exception), 'dnf is required by this method and is not installed.')
+
+
+class TestGetKojiSession(unittest.TestCase):
+    """
+    This class tests the get_koji_session method.
+    """
+    @mock.patch('__builtin__.open', create=True)
+    @mock.patch('bodhi.client.bindings.ConfigParser.readfp')
+    @mock.patch("os.path.exists")
+    @mock.patch("os.path.expanduser", return_value="/home/dudemcpants/")
+    @mock.patch('bodhi.client.bindings.BodhiClient._load_cookies')
+    @mock.patch('bodhi.client.bindings.koji.ClientSession')
+    @mock.patch('bodhi.client.bindings.ConfigParser.get')
+    def test_koji_conf_in_home_directory(self, get, koji, cookies,
+                                         expanduser, exists, readfp, mock_open):
+        """Test that if ~/.koji/config exists, we read the config from there first"""
+
+        client = bindings.BodhiClient()
+        exists.return_value = True
+        client.get_koji_session()
+
+        exists.assert_called_once_with("/home/dudemcpants/.koji/config")
+        mock_open.assert_called_once_with("/home/dudemcpants/.koji/config")
+
+    @mock.patch('__builtin__.open', create=True)
+    @mock.patch('bodhi.client.bindings.ConfigParser.readfp')
+    @mock.patch("os.path.exists")
+    @mock.patch("os.path.expanduser", return_value="/home/dudemcpants/")
+    @mock.patch('bodhi.client.bindings.BodhiClient._load_cookies')
+    @mock.patch('bodhi.client.bindings.koji.ClientSession')
+    @mock.patch('bodhi.client.bindings.ConfigParser.get')
+    def test_koji_conf_not_in_home_directory(self, get, koji, cookies,
+                                             expanduser, exists, readfp, mock_open):
+        """Test that if ~/.koji/config exists, we read the config from there first"""
```

Comment:
Maybe update this docblock to say "Test that if ~/.koji.config doesn't exist, we read config from /etc/koji.conf".